sdk/communication/Azure.Communication.Identity/README.md

```diff
@@ -11,10 +11,11 @@ Azure Communication Identity is managing tokens for Azure Communication Services
 
 ### Install the package
 
-Install the Azure Communication Identity client library for .NET with [NuGet][nuget]:
+Install the [private preview version of Azure Communication Identity client library][IdentitySDKTurnPrivatePreview] for .NET with [NuGet][nuget]:
 
 ```Powershell
-dotnet add package Azure.Communication.Identity --version 1.0.0-beta.4
+dotnet add package Azure.Communication.Common --version 1.0.0-beta.4
```

Comment:
Why do they need to explicitly import Common?

Reply:
I guess they don't need to. Nuget should pull any dependent packages. I will remove it.